packages/client/test/browser/resend.js

```diff
@@ -55,6 +57,10 @@ describe('StreamrClient Resend', () => {
             .verify.containsText('#result', '{"msg":9}')
             .assert.containsText('#result', '[{"msg":0},{"msg":1},{"msg":2},{"msg":3},{"msg":4},{"msg":5},{"msg":6},{"msg":7},{"msg":8},{"msg":9}]')
             .assert.not.elementPresent('.error')
+
+        // Wait for published messages to arrive at the storage node
+        await wait(2000)
```

Comment:
Polling the storage node here would make for a more reliable solution w.rt. time and flakyness.

Do you have direct access to `streamrClient` here? Then something like could be used:
```ts
await streamrClient.waitForStorage(lastStreamMessage)
```

If not, is it possible to push the resend button multiple times in this test until a condition is true?

Reply:
It should be possible to wait for this, I'll give it a try

Reply:
This seems a bit too workload heavy to implement given the current test setup as opposed to just just waiting. It would be good to consider replacing the client browser-tests with jest mockups similarly to the network-layer.